### DISCUSSION TITLE
Introduction - Arnold

### DISCUSSION THREAD

**Discussion by ArnoldHubach** [MyFirstBitcoin/.github#23] [Educators]
Hey all!

Arnold here, a Dutch bitcoin cheerleader. I've been working in the space for a couple of years now, mainly focusing on creating bitcoin-only educational resources.

Think about things like: co-founding www.bitcoinfocus.nl, a Dutch newsletter, pay-per-view publication & magazine, copywriting for various media outlets, creating documentaries about circular economies, visiting bitcoin communities worldwise and report about this and translating books from English to Dutch, including the following: The Bitcoin Standard, The Fiat Standard, Price of Tomorrow, Principles of Economics and Thank God for Bitcoin. 

Let's change the world together!

**Comment by Destinysmart** [comment]
**Thank God for Bitcoin** 

Great works Arnold 🫡, never knew you were a copywriter!

Destiny from Nigeria 🇳🇬
